src/main/kotlin/no/skatteetaten/aurora/boober/mapper/platform/ApplicationPlatformHandler.kt

```diff
@@ -22,11 +22,11 @@ import java.time.Duration
 abstract class ApplicationPlatformHandler(val name:String) {
     open fun handlers(handlers: Set<AuroraConfigFieldHandler>): Set<AuroraConfigFieldHandler> = handlers
 
-    abstract fun handleAuroraDeployment(auroraDeploymentSpec: AuroraDeploymentSpec, labels: Map<String, String>, mounts: List<Mount>?): AuroraDeployment
+    abstract fun handleAuroraDeployment(auroraDeploymentSpec: AuroraDeploymentSpec, labels: Map<String, String>, mounts: List<Mount>?, sidecarContainers: List<AuroraContainer>?): AuroraDeployment
 
     fun createEnvVars(mounts: List<Mount>?, auroraDeploymentSpec: AuroraDeploymentSpec): Map<String, String> {
 
-        val mountEnv = mounts?.filter { it.targetContainer == "main" }?.map {
+        val mountEnv = mounts?.filter { it.targetContainer == null }?.map {
```

Comment:
litt pirk, men kan du ikke her si !it.targetContainer

Reply:
Kan man det? Virker som man kun kan bruke not-operator på boolean uttrykk.

Reply:
Stemmer vel det ja, Det er fredag idag...